README.md

```diff
@@ -252,6 +256,14 @@ Linux amd64 Tags
 | `3.0-slim`       |  Debian 10  |
 | `3.0-appservice` |  Debian 10  |
 
+#### MCR Docs
+
+For new images update the following:
+
+- [MCR Syndication](https://github.com/microsoft/mcr/blob/main/teams/azurefunctions/azure-functions.yml)
```

Comment:
Both of these links only contain information for PowerShell 6 and 7 images. Should these include PowerShell 7.2 content as well?

Reply:
PR in progress https://github.com/microsoft/mcrdocs/pull/1063/files

Reply:
Should we remove references to Powershell 6/7 for v3 ?  Another engineer told me that the repo above is internal only so I don't know that its necessary but, we do not support powershell on v3 in GA so it may make sense to remove them or atleast note that

Reply:
Removed references for Powershell 6 for V4. Leaving V3 powershell 6 as custom containers is supported.